<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Salarios</class>
 <widget class="QMainWindow" name="Salarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Salarios</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconwindows.qrc">
    <normaloff>:/new/prefix1/Recursos/klipartz.com (4).png</normaloff>:/new/prefix1/Recursos/klipartz.com (4).png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>331</width>
         <height>78</height>
        </rect>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>11</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre: </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inNombre">
        <property name="geometry">
         <rect>
          <x>61</x>
          <y>11</y>
          <width>261</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Ingrese el nombre aquí</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>42</y>
          <width>42</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Horas: </string>
        </property>
       </widget>
       <widget class="QSpinBox" name="inHoras">
        <property name="geometry">
         <rect>
          <x>61</x>
          <y>42</y>
          <width>261</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>96</y>
         <width>321</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Jornada</string>
       </property>
       <widget class="QRadioButton" name="inMatutino">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>97</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Matutina</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="inVespertina">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>97</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Vespertina</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="inNocturna">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>97</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Nocturna</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>139</y>
         <width>339</width>
         <height>16</height>
        </rect>
       </property>
       <widget class="QPushButton" name="cmaCalcular">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;Calcular</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="cmdCalcular">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>310</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Calcular</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>0</y>
         <width>441</width>
         <height>311</height>
        </rect>
       </property>
       <widget class="QPlainTextEdit" name="outResultado">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>391</width>
          <height>271</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>Aqui se mostrará los calculos de los salarios</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>330</y>
         <width>441</width>
         <height>191</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Otros Resultados:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Salrio Bruto: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>IESS:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Salario Neto:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="outSalarioBruto">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="outIESS">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="outTotal">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="separator"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
    <addaction name="separator"/>
    <addaction name="actionSalir_2"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ay&amp;uda</string>
    </property>
    <addaction name="actioAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevo">
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>A&amp;brir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCalcular">
   <property name="text">
    <string>&amp;Calcular</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actioAcerca_de">
   <property name="text">
    <string>Ac&amp;erca de salarios</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>&amp;Calcular</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSalir_2">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconwindows.qrc"/>
 </resources>
 <connections/>
</ui>
